integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java

```diff
@@ -2,69 +2,62 @@
-                String formatTextForFullCommand = !defaultCommandChain.isPresent() || defaultCommandChain.get().endsWith(";") ? "%s%s" : "%s;%s";
-                String command = String.format(formatTextForFullCommand, defaultCommandChain.isPresent() ? defaultCommandChain : "", fileListCommand);
-                try (SSHClient sshClient = createSshClient(publicIp, sshTimeout, sshUser, privateKey)) {
-                    Pair<Integer, String> cmdOut = execute(sshClient, command);
-                    quantity += Stream.of(cmdOut.getValue().split(System.getProperty("line.separator"))).filter(s -> s != null && s.startsWith("/")).count();
-                } catch (UserAuthException | TransportException ignore) {
-                    LOGGER.warn("Whoops! SSH fail on {} while getting info for location: {}", publicIp, path);
-                }
-            }
-        }
-        return quantity;
+    @Inject
+    private CommonCloudProperties commonCloudProperties;
+
+    public String getRecipeExecutionOutput(String publicIp, String sshUser, String filePath) {
+        SSHClient sshClient = createSshClient(publicIp, sshUser);
+        return executeCommand(sshClient, "cat " + filePath);
     }
 
-    private static SSHClient createSshClient(String host, int sshTimeout, String sshUser, String defaultPrivateKeyFile) throws IOException {
-        SSHClient client = new SSHClient();
-        client.addHostKeyVerifier(new PromiscuousVerifier());
-        client.connect(host, 22);
-        client.setConnectTimeout(sshTimeout);
-        client.authPublickey(sshUser, defaultPrivateKeyFile);
-        return client;
+    private SSHClient createSshClient(String host, String sshUser) {
+        try {
+            SSHClient client = new SSHClient();
+            client.addHostKeyVerifier(new PromiscuousVerifier());
+            client.connect(host, 22);
+            client.setConnectTimeout(commonCloudProperties.getSshTimeout());
+            client.authPublickey(sshUser, commonCloudProperties.getSshPrivateKeyPath());
+            return client;
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
+        }
     }
 
-    private static Pair<Integer, String> execute(SSHClient ssh, String command) throws IOException {
+    private String executeCommand(SSHClient ssh, String command) {
         LOGGER.info("Waiting to SSH command to be executed...");
         try (Session session = startSshSession(ssh);
                     Command cmd = session.exec(command);
                     OutputStream os = IOUtils.readFully(cmd.getInputStream())) {
             LOGGER.info("The following SSH command is going to be executed on host {}: {}", ssh.getConnection().getTransport().getRemoteHost(), command);
             cmd.join(10L, TimeUnit.SECONDS);
-            return Pair.of(cmd.getExitStatus(), os.toString());
+            return os.toString();
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
```

Comment:
The same for here...

Reply:
Not needed to be logged. Exception is logged by default.